src/test/java/com/marklogic/kafka/connect/sink/MarkLogicSinkConnectorIntegrationTest.java

```diff
@@ -0,0 +1,116 @@
+package com.marklogic.kafka.connect.sink;
+
+import com.marklogic.client.io.SearchHandle;
+import com.marklogic.client.query.DeleteQueryDefinition;
+import com.marklogic.client.query.StructuredQueryBuilder;
+import com.marklogic.client.query.StructuredQueryDefinition;
+import com.marklogic.junit5.spring.SimpleTestConfig;
+import kafka.server.KafkaConfig$;
+import net.mguenther.kafka.junit.*;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+import static net.mguenther.kafka.junit.EmbeddedConnectConfig.kafkaConnect;
+import static net.mguenther.kafka.junit.EmbeddedKafkaCluster.provisionWith;
+import static net.mguenther.kafka.junit.EmbeddedKafkaClusterConfig.newClusterConfig;
+import static net.mguenther.kafka.junit.ObserveKeyValues.on;
+
+public class MarkLogicSinkConnectorIntegrationTest extends AbstractIntegrationTest {
```

Comment:
I'm thinking for these tests, we should suffix them with "KafkaTest". And for consistency with other tests (and this is my generally preferred pattern), let's do "(use case under test)KafkaTest" - so e.g. "WriteFromKafkaTest" (the lack of "WriteBatcher" or "BulkDataServices" in the name means we're using WriteBatcher).